new/t5.php

```diff
@@ -0,0 +1,13 @@
+<?php 
+$x = 1; 
+while(TRUE) {
```

Comment:
TRUE, FALSE and NULL must be lowercase; expected "true" but found "TRUE".

```suggestion
while(true) {
```